packaging/supervisord/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM centos:7
+
+ARG rpm_file=cloudify-manager-install.rpm
+ARG config=config.yaml
+ARG uid=1500
```

Comment:
Do we need to specify these UIDs?

Reply:
This is needed when mounting config file so that we can guarantee the user inside container has the enough permission to read the mounted config file

CC @tehasdf

Reply:
Ah, that's mildly icky but fair enough.